packages/teleport/src/services/user/makeAcl.ts

```diff
@@ -34,6 +34,16 @@ export default function makeAcl(json): Acl {
   const billing = json.billing || defaultAccess;
   const dbServers = json.dbServers || defaultAccess;
   const desktops = json.desktops || defaultAccess;
+  // Defaults to true, see RFD 0049
+  // https://github.com/gravitational/teleport/blob/master/rfd/0049-desktop-clipboard.md#security
+  const canShareClipboard =
+    json.clipboard !== undefined ? json.clipboard : true;
+  // Defaults to true, see RFD 0033
+  // https://github.com/gravitational/teleport/blob/master/rfd/0033-desktop-access.md#authorization
+  const desktopSessionRecording =
```

Comment:
the name `desktopSessionRecording` implies a type [Recording](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/services/recordings/types.ts#L15) that is a desktop session, i'm guessing this flag is more checking if recording is enabled for desktop sessions? then i think a better name is `XXXEnabled`, which mean i'd rename`canShareClipboard` to `clipboardEnabled`?